packages/shorebird_cli/lib/src/commands/patch/patch_aar_command.dart

```diff
@@ -52,9 +52,17 @@ of the Android app that is using this module.''',
         mandatory: true,
       )
       ..addFlag(
-        'force',
-        abbr: 'f',
-        help: 'Patch without confirmation if there are no errors.',
+        'allow-native-diffs',
+        help: '''
+Patch even if native code diffs are detected.
+NOTE: this is **not** recommended.''',
+        negatable: false,
+      )
+      ..addFlag(
+        'allow-asset-diffs',
+        help: '''
+Patch even if asset diffs are detected
+NOTE: this is **not** recommended.''',
```

Comment:
This one is less scary than the native one.  We might want to give context on why or link to docs?

Reply:
Added more info